dev/com.ibm.ws.jdbc/src/com/ibm/ws/rsadapter/impl/WSManagedConnectionFactoryImpl.java

```diff
@@ -1349,7 +1352,8 @@ public Void run() throws SQLException {
                         if(!Driver.class.equals(type)) {
                             ((CommonDataSource) dataSourceOrDriver).setLogWriter(out);
                         } else {
-                            //TODO behavior for java.sql.Driver
+                            if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled())
+                                Tr.debug(this, tc, "Unable to set logwriter on Driver type");
```

Comment:
What is the consequence of ignoring this?

Reply:
We lose out on some of the JDBC Driver logging, but there isn't really an alternative I can think of.  I did mean to test what we have when logging is enabled (I added it to the test bucket), but I forgot to check that output.

Reply:
I think we are OK but I've added it to my TODOs to ensure we are OK in all scenarios.